node/src/browserstack_local_manager.ts

```diff
@@ -1,21 +1,22 @@
 import * as browserStack from 'browserstack-local'
 import { promisify } from 'util'
 import { ConfigOptions } from 'karma'
-import { getBrowserStackAccessKey } from './browserstack_helpers'
+import { BrowserStackCredentials } from './browserstack_helpers'
 import { Logger } from './karma_logger'
 
 export class BrowserStackLocalManager {
   isRunning = false
   bsLocal: browserStack.Local = new browserStack.Local()
   switchPromise = Promise.resolve()
```

Comment:
Please make these class fields private so that you don't have to support their backward compatibility